src/mscorlib/src/System/Type.cs

```diff
@@ -1182,7 +1182,7 @@ public bool IsArray
             get { return IsArrayImpl(); }
         }
 
-        internal virtual bool IsSzArray
+        public virtual bool IsSZArray
         {
             [Pure]
             get { return false; }
```

Comment:
The API was approved based on the "throw NotImplemented" behavior. If a third party type fails to override this, we want to get a clear exception to that effect, not an assumed answer that will be wrong 10% of the time.